docs/patterns/_code-snippets.md

```diff
@@ -0,0 +1,35 @@
+---
+collection: patterns
+title: code snippets
+---
+
+## .code-snippet
+
+Code snippets should be used when presenting the user with a small snippet of code that they will likely want to copy and paste.
+
+```html
+<div class="code-snippet">
+    <input class="code-snippet__input" value="sudo apt-get update" readonly="readonly">
+    <button class="code-snippet__copy-button">Copy to clipboard</button>
+</div>
+```
+
+<div class="code-snippet">
+    <input class="code-snippet__input" value="sudo apt-get update" readonly="readonly">
+    <button class="code-snippet__action">Copy to clipboard</button>
+</div>
+
+## .code-snippet--inverse
+
+
+```html
+<div class="code-snippet code-snippet--inverse">
```

Comment:
Is this the decided method of adding modifications? Instead of a single `code-snippet--inverse` which extends `code-snippet`?